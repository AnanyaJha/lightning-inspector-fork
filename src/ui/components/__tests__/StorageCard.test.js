import React from 'react';
import StorageCard from "../StorageCard.js";
import renderer from "react-test-renderer";

/**
 * Storage Card test plan

 * 1. Verify that the card renders as expected.
 * 2. ?
 */

test("Storage Card Snapshot", () => {
    const component = renderer.create(<StorageCard storageId="__storageId__" isExpanded={true} contents="[]"/>);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});