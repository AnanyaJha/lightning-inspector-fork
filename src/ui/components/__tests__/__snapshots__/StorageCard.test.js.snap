// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storage Card Snapshot 1`] = `
<div
  className="slds-section slds-is-open slds-p-horizontal_x-small"
>
  <h3
    className="slds-section__title"
  >
    <button
      aria-controls="__storageId__"
      aria-expanded={true}
      className="slds-button slds-button_neutral slds-section__title-action"
      href={null}
      onClick={[Function]}
      title="__storageId__"
    >
      <svg
        aria-hidden="true"
        className="slds-button__icon slds-button__icon_left"
      >
        <use
          xlinkHref="/dist/slds/icons/utility-sprite/svg/symbols.svg#chevrondown"
        />
      </svg>
      <span
        className="slds-truncate"
      >
        __storageId__
      </span>
    </button>
  </h3>
  <div
    aria-hidden={true}
    className="slds-section__content"
    id="__storageId__"
  >
    <aurainspector-json
      expandTo="2"
    >
      "[]"
    </aurainspector-json>
  </div>
</div>
`;
